src/main/java/nl/tudelft/semgroup4/resources/Resources.java

```diff
@@ -123,101 +38,169 @@ public static void init() throws SlickException {
+                        int startIndex = fileString.indexOf("music");
+                        String key = fileString.substring(startIndex);
+                        
+                        Music img = new Music(fileString);
+                        music.put(key, img);
+    
+                    } catch (Exception e) {
+                        e.printStackTrace();
+                    }
+                }
+            });
+    }
 
-        titleScreenBackground = new Image("src/main/resources/img/titleScreen2.png");
-        backgroundImage = new Image("src/main/resources/img/level1.jpg");
-
-        dashboardPlayerContainerLeft = new Image(
-                "src/main/resources/img/dashboard/player_container_1.png");
-        dashboardPlayerContainerRight = new Image(
-                "src/main/resources/img/dashboard/player_container_2.png");
-        dashboardLivesContainer = new Image("src/main/resources/img/dashboard/lives_container.png");
-        dashboardlivesFull = new Image("src/main/resources/img/dashboard/lives_full.png");
-        dashboardlivesEmpty = new Image("src/main/resources/img/dashboard/lives_empty.png");
-        levelContainer = new Image("src/main/resources/img/dashboard/level_container.png");
-
-        on = new Image("src/main/resources/img/on.png");
-        off = new Image("src/main/resources/img/off.png");
-        soundText = new Image("src/main/resources/img/sound.png");
-        optionsText = new Image("src/main/resources/img/options.png");
-        backText = new Image("src/main/resources/img/back.png");
-        newKeyText = new Image("src/main/resources/img/newKeyText.png");
-        loggerText = new Image("src/main/resources/img/logger.png");
-        keyBindingsText = new Image("src/main/resources/img/keyBindings.png");
-                
-        pauseText = new Image("src/main/resources/img/pausedText.png");
-        quitText = new Image("src/main/resources/img/quitText.png");
+    /**
+     * Method that loops over all the files in the animation folder, and makes an constructs a Slick
+     * animation from the images in each inner folder, storing the animation in a hashmap, with the
+     * filepath as key.
+     * 
+     * @throws IOException
+     *             : java exception.
+     */
+    private static void loadAnimations() throws IOException {
+        HashMap<String, HashMap<String, Image>> animationImages = new HashMap<>();
+
+        Files.walk(Paths.get("src/main/resources/animation")).forEach(filePath -> {
+                if (Files.isRegularFile(filePath)) {
+                    try {
+                        String fileString = filePath.toString();
+                        int startIndex = fileString.indexOf("animation");
+                        String key = fileString.substring(startIndex);
+                        
+                        // Split the filePath with backslash as delimiter
+                        String[] layers = key.split("\\\\");
```

Comment:
Why not use System.getProperty('file.separator')? This should return a forward or back slash, based on the OS.

< bug